android/src/main/java/com/rudderstack/android/ConfigurationAndroid.kt

```diff
@@ -99,21 +97,79 @@ interface ConfigurationAndroid : Configuration {
             isPeriodicFlushEnabled: Boolean = Defaults.IS_PERIODIC_FLUSH_ENABLED,
             autoCollectAdvertId: Boolean = Defaults.AUTO_COLLECT_ADVERT_ID,
             multiProcessEnabled: Boolean = Defaults.MULTI_PROCESS_ENABLED,
-            defaultProcessName: String?= Defaults.DEFAULT_PROCESS_NAME,
+            defaultProcessName: String? = Defaults.DEFAULT_PROCESS_NAME,
             advertisingId: String? = null,
             deviceToken: String? = null,
-            logger: Logger = AndroidLogger,
+            logLevel: RudderLogger.LogLevel = Defaults.LOG_LEVEL,
```

Comment:
I believe we need to keep the default LOG_LEVEL in a different place, maybe inside the Logger interface, so that we can have a unified default log level.